sumator/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// Sumator contains two integers that we will use to sum
+type Sumator struct {
+	a int
+	b int
+}
+
+// IntSum sums two integers and returns them
+func (s Sumator) IntSum() int {
+	return s.a + s.b
+}
+
+// FloatSum sums two integers and returns a float64 number
+func (s Sumator) FloatSum() float64 {
+	f := float64(s.a + s.b)
+	return f
+}
+
+// StringSum sums two integers and returns a string
+func (s Sumator) StringSum() string {
```

Comment:
I want to add one new method `StringSum("5", "6")` and this method to return 11.